pkg/kubernetes/client_test.go

```diff
@@ -553,3 +556,50 @@ var _ = Describe("Test Namespace KubeController Methods", func() {
 	})
 
 })
+
+func TestGetEndpoint(t *testing.T) {
+	assert := tassert.New(t)
+
+	// Create endpoint controller
```

Comment:
nit: the code below isn't creating an endpoint controller, can we update the comment

Reply:
Updated!